pkgs/development/tools/buf/default.nix

```diff
@@ -19,6 +19,7 @@ buildGoModule rec {
   vendorSha256 = "sha256-HT0dsGniBoQW2Y7MhahDeFvE0nCASoPdzHETju0JuRY=";
 
   patches = [
+    # Skip a test that requires networking to be available to work.
```

Comment:
This adds 0 new information. It's a good general rule, but here the filename says it all and needs no further explanation.

Reply:
I completely agree. I want to avoid the noise of having the same 'suggestion' over and over on reviews like https://github.com/NixOS/nixpkgs/pull/130091#issuecomment-879101508, though...

Reply:
Maybe we could standardize the comment and/or filename to avoid this impasse.

Reply:
or perhaps we should not warn about 'local' patches, since we are in control of giving them meaningful names anyway?

Reply:
Reported: https://github.com/Mic92/nixpkgs-review/issues/207